deploy/olm-catalog/keycloak-operator/ci.yaml

```diff
@@ -1,3 +1,6 @@
 ---
 # Use `replaces-mode` or `semver-mode`. Once you switch to `semver-mode`, there is no easy way back.
-updateGraph: replaces-mode
\ No newline at end of file
+updateGraph: replaces-mode
+reviewers:
+  - slaskawi
```

Comment:
@vmuzikar I believe it's safe to remove Seba from the list of reviewers. If you would like to, please include me as the second reviewer.

Reply:
@abstractj This is from the times of 13.0.0 when he was still around. I'll create PR to the OperatorHub then to update the reviewers and I'll update this PR afterwards.

Reply:
I'm still here! :)

Reply:
@slaskawi yes, you are and I appreciate that. But we need to update the list of maintainers, considering your limited time.

Reply:
@abstractj Updated the reviewers. Related PRs:
* https://github.com/operator-framework/community-operators/pull/4229
* https://github.com/operator-framework/community-operators/pull/4230